nvflare/app_common/executors/in_process_client_api_executor.py

```diff
@@ -45,7 +45,8 @@
 class InProcessClientAPIExecutor(Executor):
     def __init__(
         self,
-        task_fn_path: str,
+        task_script_path: str,
+        task_script_args: str = "",
         task_fn_args: Dict = None,
```

Comment:
Since InProcessClientAPIExecutor requires `task_script_path.endswith(".py")` and uses `task_script_args`, we no longer need `task_fn_args` in this class